ftp/src/main/scala/akka/stream/alpakka/ftp/impl/FtpLike.scala

```diff
@@ -25,7 +25,7 @@ protected[ftp] trait FtpLike[FtpClient, S <: RemoteFileSettings] {
 
   def retrieveFileInputStream(name: String, handler: Handler): Try[InputStream]
 
-  def storeFileStream(name: String, handler: Handler): Try[OutputStream]
+  def storeFileStream(name: String, handler: Handler, append: Boolean): Try[OutputStream]
```

Comment:
Maybe this method should be named `storeFileOutputStream` by analogy with `retrieveFileInputStream`.